Python/btax/__init__.py

```diff
@@ -0,0 +1,2 @@
+def joke():
```

Comment:
Yes, that's from the tutorial I was a following. Is there a way to edit code in a pull request?